torch/lib/c10d/test/ProcessGroupGlooTest.cpp

```diff
@@ -185,9 +185,7 @@ void testAllreduce(const std::string& path, const at::DeviceType b) {
     auto& tensor = outputs[i][0];
     auto data = tensor.data_ptr<float>();
     for (auto j = 0; j < tensor.numel(); j++) {
-      if (data[j] != expected) {
-        throw std::runtime_error("BOOM!");
-      }
+      ASSERT_EQ(data[j], expected);
```

Comment:
Curious: does it make sense to use EXPECT_EQ() instead?

(ASSERT_ is fine too, though I've typically seen EXPECT_ preferred when the error doesn't affect the program control flow, so we can continue and see the other errors).

Reply:
But, to be clear, the change LGTM as-is.

Reply:
Makes sense, I'll change it to expect.